src/ui/panels/clan.js

```diff
@@ -39,6 +39,8 @@ export class Clan {
             name: "clans",
             x: 535,
             y: 280,
+            textStartOffsetX: 30,
+            textStartOffsetY: 0,
```

Comment:
This seems to be 30 & 0 most of the time. Maybe make those the default values and use it when the user doesn't pass them in so you don't have to pass these every time you make a scroll window w/ text in it.

Maybe something like:
```js
if (format.textStartOffsetX == undefined) {
   format.textStartOffsetX = 30;
}
```

Reply:
Updated! https://github.com/EthanVieira/rs-clicker/pull/62/commits/48f5a01bca18681a5c3316b66b087033ee369f1b